DSharpPlus.Interactivity/Extensions/ClientExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DSharpPlus.Interactivity.Extensions
+{
+    /// <summary>
+    /// Interactivity extension methods for <see cref="DiscordClient"/> and <see cref="DiscordShardedClient"/>.
+    /// </summary>
+    public static class ClientExtensions
+    {
+        /// <summary>
+        /// Enables interactivity for this <see cref="DiscordClient"/> instance.
+        /// </summary>
+        /// <param name="client">The client to enable interactivity for.</param>
+        /// <param name="configuration">A configuration instance. Default configuration values will be used if none is provided.</param>
+        /// <returns>A brand new <see cref="InteractivityExtension"/> instance.</returns>
+        /// <exception cref="InvalidOperationException">Thrown if interactivity has already been enabled for the client instance.</exception>
+        public static InteractivityExtension UseInteractivity(this DiscordClient client, InteractivityConfiguration configuration = null)
+        {
+            if (client.GetExtension<InteractivityExtension>() != null) throw new InvalidOperationException("Interactivity is already enabled for this client.");
```

Comment:
```suggestion
            if (client.GetExtension<InteractivityExtension>() != null) throw new InvalidOperationException("Interactivity is already enabled for this {(client._isShard ? "shard" : "client")}.");
```

Reply:
To keep things consistent.

Reply:
Whoops, I intended to go back and do that. Thanks for catching this!